backend/services/esweService.ts

```diff
@@ -344,4 +354,42 @@ export default class EsweService {
     }
     return createFlaggedContent(null)
   }
+
+  async getCurrentWork(nomisId: string): Promise<CurrentWork> {
+    if (!app.esweEnabled) {
+      return createFlaggedContent(null)
+    }
+
+    try {
+      const context = await this.systemOauthClient.getClientCredentialsTokens()
+      const threeMonthsAgo = moment().subtract(3, 'months').format('YYYY-MM-DD')
+      const workHistory = await this.prisonApi.getOffenderWorkHistory(context, nomisId, threeMonthsAgo)
```

Comment:
This isn't going to work I'm afraid.. the current work and work history Prison API endpoints contains the logic to decide what is current, and it is complicated.

We could modify the work history endpoint to add a flag to the return values to indicate it is current.

That's a problem for doing this PR though...
```